app/api/models/errors/CommonMtdErrors.scala

```diff
@@ -18,6 +18,8 @@ package api.models.errors
 
 import play.api.http.Status._
 
+
+object GenericAPIError extends MtdError("RULE_REQUEST_CANNOT_BE_FULFILLED","Custom (will vary depending on the actual error)",422)
```

Comment:
Can GenericAPIError be moved down to be with other rule errors (section starts at line 69), preferably at the end of the other errors

Reply:
also can it be called RuleRequestCannotBeFulfilledError to match the naming convention.
Can change the default message to the one suggested above, + ideally use it in RequestHandler so the message text isn't duplicated

Reply:
Sorted